pkg/api/component.go

```diff
@@ -54,3 +56,8 @@ type ForwardedPort struct {
 	LocalPort     int    `json:"localPort"`
 	ContainerPort int    `json:"containerPort"`
 }
+
+type Host struct {
+	Name  string   `json:"name"`
+	Hosts []string `json:"hosts"`
```

Comment:
A suggestion to omit empty hosts (might happen if ingress has no rules).

```suggestion
	Hosts []string `json:"hosts,omitempty"`
```